pose-detection/src/calculators/keypoints_smoothing.ts

```diff
@@ -67,7 +67,7 @@ export class KeypointsSmoothingFilter {
     }
 
     const objectScale =
-        objectScaleROI != null ? getObjectScale(objectScaleROI) : 1;
+        objectScaleROI != null ? getObjectScale(objectScaleROI, imageSize) : 1;
```

Comment:
Looks like it changes logic for regular landmarks as well right?
If true - move to separate CL, cause it's a pretty significant change and is lost among other changes in this CL.

Reply:
This change was temporary and is now gone in the newer commits